__tests__/endpoints.test.js

```diff
@@ -244,6 +244,23 @@ describe("/api/users", () => {
         })
     })
 })
+describe("/api/articles?query", () => {
+    test('200 with all an array of articles filtered by the query value(specific topic)', () => {
+        return request(app).get('/api/articles?topic=cats').expect(200)
+        .then((res) => {
+            const articles = res.body
+            expect(articles.length).toBe(1)
+            expect(articles[0].topic).toBe("cats")
+        })
+    })
```

Comment:
One test missing here: We have a valid topic with no associated articles (paper). In this case we would expect 200 responds with an empty array